package.json

```diff
@@ -93,6 +92,11 @@
         "url-polyfill": "^1.1.10",
         "whatwg-fetch": "^3.4.0"
     },
+    "peerDependencies": {
+        "react": "^16.13.1",
+        "react-dom": "^16.13.1",
+        "styled-jsx": "^3.3.0"
+    },
```

Comment:
I don't think these should be peerDependencies, since this is the top-level app.  There may be some straggling documentation somewhere which suggests using peerDeps, but regular dependencies work fine!

Reply:
Read it here: https://ui.dhis2.nu/#/getting-started
Will fix!